### ISSUE TITLE
Update you keep talking about

### ISSUE THREAD

**Issue by piepie62** [Vhati/ftl-profile-editor#103]
Hey, I've been ghosting this for a while, and you have talked about releasing an update multiple times. Just wondering, is it actually done? If not, that's fine, it needs work, but if it is could you push the update? Thanks for your work. :)

**Comment by Vhati** [comment]
Yeah, you're right. I'd drifted away from this. Plus Java 1.9's gonna break the editor altogether unless I tweak the launcher scripts/executable. I'm working on this tonight. Thanks for the persistence. :)

**Comment by piepie62** [comment]
Thank you for the work you have done. I cannot imagine that reverse engineering is fun.
